src/scenes/level.js

```diff
@@ -252,6 +273,53 @@ export class LevelScene extends Phaser.Scene {
             this.inventory.button.setAlpha(0.1);
         });
 
+        // Skills
+        this.skills.panel = this.add
+            .image(548, 208, "skills-panel")
+            .setOrigin(0, 0)
+            .setDepth(1);
+        this.skills.button = this.add
+            .image(560, 168, "skills-button")
+            .setOrigin(0, 0)
+            .setDepth(2)
+            .setInteractive();
+        this.skills.button.on("pointerup", () => {
+            this.showSkills(true);
+        });
+
+        // Skills text
+        this.attackText = this.add
```

Comment:
These are defaulted and shown whenever you move to a different area. They aren't updated until an enemy is clicked - it is never updated in woodcutting scene. These should display what is in character data and not defaulted to 1s.

Reply:
oops

Also I haven't done woodcutting yet, I was just prioritizing combat, wouldn't be too hard to add the text for this though

Reply:
Added the text and xp